Bio/Blast/ParseBlastTable.py

```diff
@@ -53,6 +57,11 @@ def add_entry(self, entry):
 
 
 class BlastTableReader(object):
+    """Parse the output of blastpgp.
```

Comment:
Here, `Parser for ...` or `Reader for ...` seems better. Perhaps, also remove the `Return a ...`, since it's the method of the class (rather than the class itself) that returns the value.

Reply:
I wasn't sure whether it would pass "It prescribes the function or method's effect as a command" of PEP257, but they all did, great.